### PR TITLE
Watchify and css modules

### PR DESCRIPTION
**Pull request by gabrielpoca**

### CODE REVIEW

File: example/Procfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+styleguide: rsg 'components/**/*.js' -w
```
Comment (azevedo-252) [inline example/Procfile]: what about .jsx?

File: lib/rsg.js [hunk 2]
```diff
@@ -48,7 +49,11 @@ function RSG (input, opts) {
   opts.pushstate = opts.pushstate || false
   opts.files = opts.files || []
   opts.babelConfig = opts.babelConfig || null
-  opts.browserifyConfig = assign({ debug: true }, opts.browserifyConfig, { standalone: 'Contents' })
+  opts.browserifyConfig = assign({ debug: true }, opts.browserifyConfig, {
+    cache: {},
```
Comment (azevedo-252) [inline lib/rsg.js]: Why these two new options?

File: lib/rsg.js [hunk 3]
```diff
@@ -314,34 +324,49 @@ RSG.prototype.createContentsFile = function () {
...
+          resolve()
+        })
 
-    b = watchify(b)
+      }).on('data', function() {});
 
-    b.on('update', function () {
-      self.log.info('styleguide updating')
+      bundle.on('update', function () {
+        console.log('styleguide updateding')
```
Comment (azevedo-252) [inline lib/rsg.js]: I know it was already there, but: typo ;)

File: example/Procfile (hunk 1 above)
Comment (gabrielpoca) [inline example/Procfile]: I believe it doesn't matter the extension to use jsx. I've only matched the existing setup.

File: lib/rsg.js (hunk 2 above)
Comment (gabrielpoca) [inline lib/rsg.js]: Required by watchify.

File: lib/rsg.js (hunk 3 above)
Comment (gabrielpoca) [inline lib/rsg.js]: Fixed.

**Comment by azevedo-252** [comment]
:+1:

**Pull request closed without merging**